README.md

```diff
@@ -87,8 +92,8 @@ This modules ships the Fargate Continaer logs to CloudWatch
 
 # How to Deploy
 
-## Pre-requisites:
-Ensure that you installed the following tools in your Mac or Windows Laptop before start working with this module and run Terraform Plan and Apply
+## Prerequisites:
+Ensure that you have installed the following tools in your Mac or Windows Laptop before start working with this module and run Terraform Plan and Appl
```

Comment:
Typo at the end. Should this be "Apply"?

Reply:
Oops, sorry about that, fixed!